data_model/src/query/mod.rs

```diff
@@ -176,7 +176,7 @@ pub mod model {
 
     /// Query with parameters client can specify.
     #[derive(
-        Debug, Constructor, Getters, Clone, PartialEq, Eq, Encode, Decode, Serialize, Deserialize,
+        Clone, Debug, PartialEq, Eq, Constructor, Getters, Encode, Decode, Serialize, Deserialize,
```

Comment:
So is our policy to place std derives first? Didn't know about that

Reply:
there is no strict policy. If you look around you will see this pattern